cases/missing-secret-ref/deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+        - name: nginx
+          image: nginx:latest
+          envFrom:
+            - secretRef:
+                name: missing-secret
```

Comment:
don't name it missing-secret because the ai often sees that and uses the name to assume it's missing